packages/components/Button/src/Button.tsx

```diff
@@ -81,6 +83,13 @@ export const Button = compose<ButtonType>({
           {shouldShowIcon && iconPosition === 'after' && <Slots.icon {...iconProps} />}
         </Slots.root>
       );
+
+      const shouldShowShadow = !appearance && Platform.OS === 'macos';
+      if (shouldShowShadow) {
+        return <Slots.shadow>{buttonWithoutShadow}</Slots.shadow>;
+      } else {
+        return buttonWithoutShadow;
+      }
```

Comment:
See comment above, could we scope the change to just FAB for this PR? Reason I say this is because I'd rather have a more general way to say "if no shadow token, don't render shadow" rather than explicitly checking the platform. If one day designers are like "JK no shadows on macOS", it would be more work than a simple token change to fix Button then.

Reply:
Yep sorry just forgot to push the commit that reverts the macOS stuff